src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -472,9 +469,9 @@ dellsc_multipath_xref=true
 enable_unity_backend=false
 # The Dell EMC Unity Container is published in the RH Container Catalog
 # registry.connect.redhat.com
-# By default, the latest image of the RHOSP release is pulled.
+# By default, this version of the image of the RHOSP release is pulled.
 # Change to a different version if a specific image is required.
-cinder_unity_container_version=latest
+cinder_unity_container_version=16.1-1
```

Comment:
You might have to rebase again, this has changed to 16.1-2

Reply:
Roger that. Will do it.